experiments.tex

```diff
@@ -84,7 +85,8 @@ \subsection{Effect of $\overline{\tcomp}/\overline{\tIO}$ on Performance}
 \subsubsection{Effect of Increased Compute workload for RMSD Task}
 
 The increased computational workloads corresponded to $\overline{\tcomp}/\overline{\tIO}$ ratios of 11, 19, 27 respectively as shown in Table \ref{tab:load-ratio}.
-In fact, the $\overline{\tcomp}/\overline{\tIO}$ ratio for the higher workloads (with factor $X$) should be $X$ times the $\overline{\tcomp}/\overline{\tIO}$ for the $1\times$ workload (In agreement with theoretical prediction in Table \ref{tab:load-ratio}).
+\gpnote{I am not sure if this breaking passes the same message. It needs fixing.}
```

Comment:
What do you mean with this comment?

Reply:
There a very large sentence here and it was fixed by Mahzad. I forgot to remove the comment. Fixing!